src/lib/route-segments.test.js

```diff
@@ -0,0 +1,92 @@
+/* global it, expect */
```

Comment:
Here's a new thing I learned! `/* eslint-env jest */` will cover all [jest globals](https://github.com/sindresorhus/globals/blob/master/globals.json#L925), so you wouldn't need to specify globals manually.